before.Sentry.sln.targets

```diff
@@ -0,0 +1,8 @@
+<Project InitialTargets="RestoreSubmodule">
+  <!-- If Ben.Demystifer is not found, restore git submodules -->
+  <Target Name="RestoreSubmodule"
+    Condition="!Exists('modules/Ben.Demystifier/Ben.Demystifier.sln')">
+    <Message Importance="High" Text="Ben.Demystifer not found. Restoring git submodules."></Message>
+    <Exec Command="git submodule update --init --recursive"></Exec>
```

Comment:
```suggestion
    <Message Importance="High" Text="Ben.Demystifer not found. Restoring git submodules." />
    <Exec Command="git submodule update --init --recursive" />
```

Reply:
Worth setting working directory on `Exec`? Otherwise depending on which folder `dotnet build` is ran from, it may not work.

Reply:
You can run that from any directory actually, except `modules/Ben.Demystifier`.
But yeah it's a good idea anyway for consistency. Do you know by heart how to do that?

Reply:
The `WorkingDirectory` property: https://docs.microsoft.com/en-us/visualstudio/msbuild/exec-task?view=vs-2019

Reply:
Here's what I learned:
This file is only loaded when building the solution. So if you `cd src/Sentry` and `dotnet build` it doesn't take effect.

The value of `$(MSBuildProjectDirectory)` when building the sln was the folder where the sln was. But removing `WorkingDirectory="$(MSBuildProjectDirectory)` and building it from outside the project directly worked just fine. 

```
➜  sentry-dotnet git:(ref/restore-submodules) ✗ cd ..
➜  git dotnet build sentry-dotnet/Sentry.sln 
Microsoft (R) Build Engine version 16.11.0-preview-21254-21+e73d08c28 for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

  Ben.Demystifer not found. Restoring git submodules.
  Submodule path 'modules/Ben.Demystifier': checked out 'a6fe04f9023d7cf304e676ea2319f31b2f5fda72'
```